src/cli.js

```diff
@@ -400,17 +405,18 @@ function cli(inputArgs) {
                             , nohooks : args.nohooks
                             , cli_variables : cli_vars
                             , browserify: args.browserify || false
-                            , fetch: args.fetch || false
+                            , fetch: args.fetch
                             , link: args.link || false
                             , save: args.save
                             , shrinkwrap: args.shrinkwrap || false
                             , force: args.force || false
+
         };
         return cordova.raw[cmd](subcommand, targets, download_opts);
     }
 }
 
- function create(undashed, args) {
+    function create(undashed, args) {
```

Comment:
I think this shouldn't be tabbed in. Everything following this will need to get readjusted (remove 1 tab from each line)